modules/auth/repo_form.go

```diff
@@ -135,6 +135,8 @@ type RepoSettingForm struct {
 	PullsAllowRebase                 bool
 	PullsAllowRebaseMerge            bool
 	PullsAllowSquash                 bool
+	PullsAllowManualMerge            bool
+	EnableAutomaticManualMerge       bool
```

Comment:
Ah sorry I think AutodetectManualMerge is better!

```suggestion
	EnableAutodetectManualMerge       bool
```